src/cls/tabular/cls_tabular_utils.cc

```diff
@@ -596,6 +596,655 @@ int processSkyFb(
+                    //std::cout << "curr_col_data_type_sky = " << curr_col_data_type_sky << std::endl ;
+
+                    if (j < AGG_COL_LAST or j > col_idx_max) {
+                        errcode = TablesErrCodes::RequestedColIndexOOB;
+                        errmsg.append("ERROR processSkyFb(): table=" +
+                                root.table_name + "; rid=" +
+                                std::to_string(this_rid) + " j=" +
+                                std::to_string(j) + " OOB.");
+                    } else {
+                        switch( curr_col_data_type_sky ) {  // encode data val into flexbuf
+                            //case SDT_INT8:
+                            //    flexbldr->Add(row[col.idx].AsInt8());
+                            //    break;
+                            //case SDT_INT16:
+                            //    flexbldr->Add(row[col.idx].AsInt16());
+                            //    break;
+                            //case SDT_INT32:
+                            //    flexbldr->Add(row[col.idx].AsInt32());
+                            //    break;
+                            //case SDT_INT64:
+                            //    flexbldr->Add(row[col.idx].AsInt64());
+                            //    break;
+                            //case SDT_UINT8:
+                            //    flexbldr->Add(row[col.idx].AsUInt8());
+                            //    break;
+                            //case SDT_UINT16:
+                            //    flexbldr->Add(row[col.idx].AsUInt16());
+                            //    break;
+                            //case SDT_UINT32:
+                            //    flexbldr->Add(row[col.idx].AsUInt32());
+                            //    break;
+                            case SDT_UINT64: {
+                                auto column_of_data = 
+                                    static_cast< const Tables::SDT_UINT64_FBU* >( curr_col_data ) ;
+                                auto data_at_row = column_of_data->data()->Get( rnum ) ;
+                                //std::cout << std::to_string( data_at_row ) << std::endl ;
+                                flexbldr->Add( data_at_row );
+                                break;
+                            }
+                            //case SDT_CHAR:
+                            //    flexbldr->Add(row[col.idx].AsInt8());
+                            //    break;
+                            //case SDT_UCHAR:
+                            //    flexbldr->Add(row[col.idx].AsUInt8());
+                            //    break;
+                            //case SDT_BOOL:
+                            //    flexbldr->Add(row[col.idx].AsBool());
+                            //    break;
+                            case SDT_FLOAT: {
+                                auto column_of_data = 
+                                    static_cast< const Tables::SDT_FLOAT_FBU* >( curr_col_data ) ;
```

Comment:
In general you're using spaces around args, its not a big deal but since our other code is not using those, could you remove them at least in the cls files?  Its fine if you want to leave in your other specific files like fbuwriter etc.